go/plugins/googleai/googleai.go

```diff
@@ -140,7 +138,6 @@ func KnownModels() []string {
 	return keys
 }
 
-// requires state.mu
```

Comment:
This is still true, right? The lowercase methods require the lock to be held.

Reply:
Right. Done.